ehrapy/io/_read.py

```diff
@@ -56,7 +43,45 @@ def read(
         backup_url: URL to download the data file(s) from if not yet existing.
 
     Returns:
-        An :class:`~anndata.AnnData` object, a :class:`~mudata.MuData` object or a dict with an identifier (usually the filename, without extension)
+        An :class:`~anndata.AnnData` object, a :class:`~mudata.MuData` object or a dict with an identifier (the filename, without extension)
+        for each :class:`~anndata.AnnData` object in the dict
+
+    Example:
+        .. code-block:: python
+
+            import ehrapy as ep
+            adata = ep.io.read_cvs("myfile.csv")
+    """
+    file: Path = Path(dataset_path)
+    if not file.exists():
+        file = _get_non_existing_files(file, download_dataset_name, backup_url)
+
+    adata = _read_csv(
+        filename=file,
+        extension=extension,
```

Comment:
Maybe we can remove the "extension" parameter here now and use the Pandas parameter `sep`? More consistent API with the big packages?